modin/pandas/dataframe.py

```diff
@@ -2195,23 +2195,26 @@ def to_parquet(
         compression="snappy",
         index=None,
         partition_cols=None,
-        storage_options: StorageOptions = None,
```

Comment:
Why is this removed?

Reply:
I see that the default is None is the Pandas docs here (https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_parquet.html?highlight=to_parquet#pandas.DataFrame.to_parquet), did not understand the need for adding the `StorageOptions`

Reply:
@naren-ponder We can probably add it back. The actual implementation of pandas uses it as a type hint: https://github.com/pandas-dev/pandas/blob/v1.3.4/pandas/core/frame.py#L2586

I think it probably should be `Optional[StorageOptions]` in pandas, but they choose to make it optional all the time which is where I think the confusion is coming from: https://github.com/pandas-dev/pandas/blob/v1.3.4/pandas/_typing.py#L177

Reply:
Suggest adhering to pandas.